Panic trying to run `pulumi-tfgen-azure python` in `pulumi-azure`

Trying to update `pulumi-azure` to latest `pulumi-terraform` is hitting an issue in the Python generator.

```
$ make
BUILD:
go install -ldflags "-X github.com/pulumi/pulumi-azure/pkg/version.Version=v0.11.1-dev-1522868625-g79ecc32-dirty" github.com/pulumi/pulumi-azure/cmd/pulumi-tfgen-azure
go install -ldflags "-X github.com/pulumi/pulumi-azure/pkg/version.Version=v0.11.1-dev-1522868625-g79ecc32-dirty" github.com/pulumi/pulumi-azure/cmd/pulumi-resource-azure
for LANGUAGE in "nodejs" "python" ; do \
                pulumi-tfgen-azure $LANGUAGE --overlays overlays/$LANGUAGE/ --out pack/$LANGUAGE/ || exit 3 ; \
        done
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x38 pc=0x24610bc]

goroutine 1 [running]:
github.com/pulumi/pulumi-azure/vendor/github.com/pulumi/pulumi-terraform/pkg/tfgen.pyTypeFromSchema(0xc4203dd0e0, 0x0, 0x258a660, 0xc42068e370)
        /Users/luke/go/src/github.com/pulumi/pulumi-azure/vendor/github.com/pulumi/pulumi-terraform/pkg/tfgen/generate_python.go:664 +0x12c
github.com/pulumi/pulumi-azure/vendor/github.com/pulumi/pulumi-terraform/pkg/tfgen.pyType(0xc4208786c0, 0x5, 0xc420688d90)
        /Users/luke/go/src/github.com/pulumi/pulumi-azure/vendor/github.com/pulumi/pulumi-terraform/pkg/tfgen/generate_python.go:646 +0x38
github.com/pulumi/pulumi-azure/vendor/github.com/pulumi/pulumi-terraform/pkg/tfgen.(*pythonGenerator).emitResourceType(0xc420476000, 0xc4209218f0, 0xc42054cab0, 0x0, 0x0, 0x0, 0x0)
        /Users/luke/go/src/github.com/pulumi/pulumi-azure/vendor/github.com/pulumi/pulumi-terraform/pkg/tfgen/generate_python.go:400 +0x8eb
github.com/pulumi/pulumi-azure/vendor/github.com/pulumi/pulumi-terraform/pkg/tfgen.(*pythonGenerator).emitModuleMember(0xc420476000, 0xc4209218f0, 0x35c7ac0, 0xc42054cab0, 0x2, 0xc42035b7a0, 0x1, 0x2)
        /Users/luke/go/src/github.com/pulumi/pulumi-azure/vendor/github.com/pulumi/pulumi-terraform/pkg/tfgen/generate_python.go:213 +0x496
github.com/pulumi/pulumi-azure/vendor/github.com/pulumi/pulumi-terraform/pkg/tfgen.(*pythonGenerator).emitModule(0xc420476000, 0xc4209218f0, 0x0, 0x0, 0x0, 0x0, 0x0)
        /Users/luke/go/src/github.com/pulumi/pulumi-azure/vendor/github.com/pulumi/pulumi-terraform/pkg/tfgen/generate_python.go:146 +0x1d4
github.com/pulumi/pulumi-azure/vendor/github.com/pulumi/pulumi-terraform/pkg/tfgen.(*pythonGenerator).emitModules(0xc420476000, 0xc4204703c0, 0x0, 0x0, 0xc42099f848, 0x244face, 0x7ffeefbff904)
        /Users/luke/go/src/github.com/pulumi/pulumi-azure/vendor/github.com/pulumi/pulumi-terraform/pkg/tfgen/generate_python.go:109 +0xcf
github.com/pulumi/pulumi-azure/vendor/github.com/pulumi/pulumi-terraform/pkg/tfgen.(*pythonGenerator).emitPackage(0xc420476000, 0xc4202f5220, 0x0, 0x0)
        /Users/luke/go/src/github.com/pulumi/pulumi-azure/vendor/github.com/pulumi/pulumi-terraform/pkg/tfgen/generate_python.go:86 +0x3d
github.com/pulumi/pulumi-azure/vendor/github.com/pulumi/pulumi-terraform/pkg/tfgen.(*generator).Generate(0xc4202a0b00, 0x5, 0x2a05e40)
        /Users/luke/go/src/github.com/pulumi/pulumi-azure/vendor/github.com/pulumi/pulumi-terraform/pkg/tfgen/generate.go:337 +0x94
github.com/pulumi/pulumi-azure/vendor/github.com/pulumi/pulumi-terraform/pkg/tfgen.newTFGenCmd.func1(0xc42035e900, 0xc4202f51d0, 0x1, 0x5, 0x176a5b9, 0xc42035e900)
        /Users/luke/go/src/github.com/pulumi/pulumi-azure/vendor/github.com/pulumi/pulumi-terraform/pkg/tfgen/main.go:54 +0x123
github.com/pulumi/pulumi-azure/vendor/github.com/pulumi/pulumi/pkg/util/cmdutil.RunFunc.func1(0xc42035e900, 0xc4202f51d0, 0x1, 0x5)
        /Users/luke/go/src/github.com/pulumi/pulumi-azure/vendor/github.com/pulumi/pulumi/pkg/util/cmdutil/exit.go:55 +0x54
github.com/pulumi/pulumi-azure/vendor/github.com/spf13/cobra.(*Command).execute(0xc42035e900, 0xc4200100d0, 0x5, 0x5, 0xc42035e900, 0xc4200100d0)
        /Users/luke/go/src/github.com/pulumi/pulumi-azure/vendor/github.com/spf13/cobra/command.go:750 +0x2c1
github.com/pulumi/pulumi-azure/vendor/github.com/spf13/cobra.(*Command).ExecuteC(0xc42035e900, 0xc4201d3604, 0x2563240, 0xc4203df600)
        /Users/luke/go/src/github.com/pulumi/pulumi-azure/vendor/github.com/spf13/cobra/command.go:831 +0x30e
github.com/pulumi/pulumi-azure/vendor/github.com/spf13/cobra.(*Command).Execute(0xc42035e900, 0x5, 0x2a05e40)
        /Users/luke/go/src/github.com/pulumi/pulumi-azure/vendor/github.com/spf13/cobra/command.go:784 +0x2b
github.com/pulumi/pulumi-azure/vendor/github.com/pulumi/pulumi-terraform/pkg/tfgen.Main(0x285b043, 0x5, 0x2a05e40, 0x25, 0xc42045c380, 0x285d884, 0x7, 0x28abb40, 0x4b, 0xc4203de540, ...)
        /Users/luke/go/src/github.com/pulumi/pulumi-azure/vendor/github.com/pulumi/pulumi-terraform/pkg/tfgen/main.go:19 +0x98
main.main()
        /Users/luke/go/src/github.com/pulumi/pulumi-azure/cmd/pulumi-tfgen-azure/main.go:12 +0xa6
make: *** [build] Error 3

```

--- a/pkg/tfgen/generate_python.go
+++ b/pkg/tfgen/generate_python.go
@@ -660,8 +660,12 @@ func pyTypeFromSchema(sch *schema.Schema, info *tfbridge.SchemaInfo) string {
 		if tfbridge.IsMaxItemsOne(sch, info) {
 			// This isn't supposed to be projected as a list; project it as a scalar.
 			if elem, ok := sch.Elem.(*schema.Schema); ok {
+				var schInfo *tfbridge.SchemaInfo
+				if info != nil {
+					schInfo = info.Elem
+				}
 				// If the elem is a schema type, see if we can do better than just "dict".
-				return pyTypeFromSchema(elem, info.Elem)
+				return pyTypeFromSchema(elem, schInfo)
 			}
 			// Otherwise, return "dict".
 			return "dict"

